crazyswarm2/src/crazyswarm2_server.cpp

```diff
@@ -98,6 +100,8 @@ class CrazyflieROS
     service_upload_trajectory_ = node->create_service<UploadTrajectory>(name + "/upload_trajectory", std::bind(&CrazyflieROS::upload_trajectory, this, _1, _2));
 
     subscription_cmd_vel_ = node->create_subscription<geometry_msgs::msg::Twist>(name + "/cmd_vel", rclcpp::SystemDefaultsQoS(), std::bind(&CrazyflieROS::cmd_vel_changed, this, _1)); 
+    subscription_cmdFullState_ = node->create_subscription<crazyswarm2_interfaces::msg::FullState>(name + "/cmdFullState", rclcpp::SystemDefaultsQoS(), std::bind(&CrazyflieROS::cmdFullStateSetpoint, this, _1)); 
```

Comment:
for consistency, please rename to `cmd_full_state`. Also change the function name to `cmd_full_state_changed`.